.github/workflows/docker-build-and-push.yml

```diff
@@ -92,6 +92,7 @@ jobs:
               SEARCH_URL=${{ inputs.search_url }}
               FEED_DISCOVERY_URL=${{ inputs.feed_discovery_url }}
               STATUS_URL=${{ inputs.status_url }}
+              GIT_COMMIT=${{ github.sha }}
```

Comment:
I didn't you could use this! Neato!

Docs: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
There are so many toys in GitHub Actions, it's amazing.